.github/dependabot.yml

```diff
@@ -9,3 +9,7 @@ updates:
     directory: "/" # Location of package manifests
     schedule:
       interval: "weekly"
+    allow:
+    - dependency-name: "*"
+      dependency-type: "production"
+
```

Comment:
Implemented matching this doc https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#dependency-type-allow